src/views/ipv/page-f2f-multiple-doc-check.njk

```diff
@@ -2,53 +2,76 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% set errorState = pageErrorState %}
 {% set pageTitleName = 'pages.pageF2FMultipleDockCheck.title' | translate %}
 {% set googleTagManagerPageId = "pageF2FMultipleDockCheck" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l" id="header" data-page="{{googleTagManagerPageId}}">{{'pages.pageF2FMultipleDockCheck.header' | translate }}</h1>
-  <p class="govuk-body">{{'pages.pageF2FMultipleDockCheck.content.paragraph1' | translate | safe }}</p>
+  {% if errorState %}
```

Comment:
Can we move this block to `base.njk` and somehow pass it the text entries to display?
Maybe in the validation check create a list of errors (currently maximum length of 1 :) ) that contain the relevant bits of text and then have the shared template display the error box if there are any validation errors.

Reply:
Yes this can be moved into the base.njk, however I think it would still be best to keep the errors in the internationalisation files for consistency and makes it easier to maintain. What I can do is move it into the base and then in the html template set the necessary fields. This would reduce it to half the amount of lines in each template